frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.html

```diff
@@ -3,7 +3,7 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 <div class="dialog">
-  <div class="header">
+  <div class="header flex-col">
```

Comment:
Here the `header` class does not contain the `display: flex` attribute, so what exactly does `flex-col` does in case of a non-existing `flex`?

Update: Same as above. Dialog has a custom `header` and has no `mat-card-overview` as parent (so you can simply remove the `flex-col` here again)